packages/terra-dialog-modal/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+#Fixed
```

Comment:
This needs a space to format correctly. `# Fixed`

![image](https://user-images.githubusercontent.com/6720991/63521770-b0c35d80-c4bc-11e9-9303-510fb4074da8.png)

Reply:
formatted : 4850084c77a35d6e21b4c8c5b87c573595e5d25b